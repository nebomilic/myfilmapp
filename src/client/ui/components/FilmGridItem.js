import React from 'react';
import PropTypes from 'prop-types';

const FilmGridItem = ({title, image, rating, description}) => { 
    return (
        <div className='film-grid-item'>
            <img src={image} />
            <h4>{title}   [ {rating} ]</h4>
            <p>{description}</p>            
        </div>  

    ); 
};   

FilmGridItem.propTypes = {
    title: PropTypes.string,
    image: PropTypes.string,
    rating: PropTypes.number,
    description: PropTypes.string,
};

export default FilmGridItem;