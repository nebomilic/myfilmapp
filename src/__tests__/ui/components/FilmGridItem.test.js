import React from 'react';
import FilmGridItem from '../../../client/ui/components/FilmGridItem';
import renderer from 'react-test-renderer';

it ('renders corectly', () => {
    const tree = renderer
        .create(<FilmGridItem/>)
        .toJSON();

    expect(tree).toMatchSnapshot();
})