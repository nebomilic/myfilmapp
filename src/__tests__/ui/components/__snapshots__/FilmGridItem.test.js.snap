// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders corectly 1`] = `
<div
  className="film-grid-item"
>
  <img
    src={undefined}
  />
  <h4>
       [ 
     ]
  </h4>
  <p />
</div>
`;
